src/core/worker.js

```diff
@@ -485,6 +485,13 @@ class WorkerMessageHandler {
       return pdfManager.ensureCatalog("documentOutline");
     });
 
+    handler.on(
+      "GetOptionalContentConfig",
+      function wphSetupGetOptionalContentConfig(data) {
```

Comment:
This handler can be shortened a bit by using an anonymous function like the one below.